osquery/tables/events/darwin/openbsm_events.cpp

```diff
@@ -10,10 +10,13 @@
 #include <arpa/inet.h>
 
 #include <bsm/libbsm.h>
+#include <bsm/audit_kevents.h>
```

Comment:
What did you need from this header that we didn't have already?

Reply:
All the `AUE_*` macros.